src/anchos.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+const Web3 = require('web3');
+const Mosaic = require('@openstfoundation/mosaic.js');
+const program = require('commander');
+const fs = require('fs');
+
+const StateRootAnchorService = require('./state_root_anchor_service.js');
+
+program
+  .version('0.1.0', '-v, --version')
+  .option('-d --direction <origin | auxiliary>', 'The source chain to anchor a state root '
+  + 'to the target chain.', /^(origin|auxiliary)$/i)
+  .option('-d, --delay <block number>', 'The delay to wait in blocks on the source '
+  + 'chain before anchoring.', /^\d+$/i)
```

Comment:
at least direction and delay are arguments, not options, I would say

Reply:
I am confused. What do you mean?

Reply:
Are they optional or required?

Reply:
You may want to use arguments instead is all I'm saying.

Reply:
They are required. However, I've assumed that this is optional:
    `.option('-y --delay [delay]', '...')` -- [] square brackets
And this one is required:
    `.option('-y --delay <delay>', '...')` -- <> brackets

Reply:
`[]` denotes optional *value* to the *option* (in the below case with a default value). The option itself is still optional.
```js
  .option('-c, --cheese [type]', 'Add the specified type of cheese [marble]', 'marble')
```

Reply:
Here `dir` is a required *argument*:
```js
program
  .command('rm <dir>')
```

Reply:
True. However, using `.command` is for defining the sub-commands, yes?

Reply:
There is also `.arguments`